packages/react-native-reanimated/android/src/main/cpp/reanimated/CMakeLists.txt

```diff
@@ -55,4 +57,5 @@ set_target_properties(
 target_link_libraries(
     reanimated
     worklets
+    ReactAndroid::react_nativemodule_core
```

Comment:
This change is okay but I would expect it to be closer to other `ReactAndroid::*` prefabs. I guess this one is not required for `worklets`, is that correct?